.github/workflows/java-ci.yml

```diff
@@ -0,0 +1,26 @@
+# This workflow will build a Java project with Maven
```

Comment:
Failed to execute goal org.jacoco:jacoco-maven-plugin:0.8.6:prepare-agent (default) on project my-home-supplies: Execution default of goal org.jacoco:jacoco-maven-plugin:0.8.6:prepare-agent failed: Plugin org.jacoco:jacoco-maven-plugin:0.8.6 or one of its dependencies could not be resolved: Could not transfer artifact org.apache.maven:maven-artifact:jar:3.0 from/to central (https://repo.maven.apache.org/maven2): Transfer failed for https://repo.maven.apache.org/maven2/org/apache/maven/maven-artifact/3.0/maven-artifact-3.0.jar: Connection reset -> [Help 1]